lms/templates/commerce/provider.underscore

```diff
@@ -1,14 +1,14 @@
 <div class="provider-wrapper">
   <div class="provider-info">
-    <%- interpolate(
+    <%- edx.HtmlUtils.interpolateHtml(
           gettext("You still need to visit the %(display_name)s website to complete the credit process."),
-          { display_name: display_name }, true
+          { display_name: display_name }
         ) %>
   </div>
   <div class="provider-more-info">
-    <%- interpolate(
+    <%- edx.HtmlUtils.interpolateHtml(
           gettext("To finalize course credit, %(display_name)s requires %(platform_name)s learners to submit a credit request."),
```

Comment:
FYI: @andy-armstrong: I added TNL-4775: Lint for %(...)s in HtmlUtils.interpolateHtml() or StringUtils.interpolate() calls